ci-operator/step-registry/redhat-chaos/cerberus/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - chaitanyaenr
+  - yogananth-subramanian
+  - paigerube14
+  - tsebastiani
+reviewers:
```

Comment:
@paigerube14 we might want to add you and me to the reviewers list as well. Thoughts?

Reply:
I think approvers auto get "reviewer" access as well. I'll read up on it to be sure though